src/data-fetcher/bigwig/higlass-bigwig-datafetcher.ts

```diff
@@ -0,0 +1,323 @@
+/* TODO: Support esm
```

Comment:
Not sure what support ESM means here? This JavaScript file uses the `import` & `export` syntax, which is what I assume is meant by ESM.